streams/shared/src/main/scala/zio/stream/experimental/ZPipeline.scala

```diff
@@ -273,6 +276,9 @@ object ZPipeline extends ZPipelineCompanionVersionSpecific {
         stream
     }
 
+  def iso_8859_1Decode: UtfDecodingPipeline =
```

Comment:
I understand why you did it but I think this type alias may ultimately make things less accessible when we have a lot of different pipelines with different types. I would suggest just going back to the normal type parameters.

Reply:
Will remove this type alia.